int64 usia
float64 tinggi
string nama
bool kehadiran
float64 komponenX
float64 komponenY
float64 komponenZ
float64 magnitude